tests/class.tx_mksearch_tests_DbTestcase.php

```diff
@@ -98,6 +98,8 @@ public function __construct($name = null, array $data = array(), $dataName = '')
      */
     protected function setUp()
     {
+        self::markTestIncomplete("some TCA Problem");
```

Comment:
Please change message. Better would be something like this: Database tests are no longer supported. Please switch to functional tests

Reply:
changed it